Code/Tools/ProjectManager/Source/GemCatalog/GemInspector.h

```diff
@@ -44,10 +44,16 @@ namespace O3DE::ProjectManager
         inline constexpr static const char* s_headerColor = "#FFFFFF";
         inline constexpr static const char* s_textColor = "#DDDDDD";
 
+        QModelIndex GetCurrentModelIndex()
+        {
+            return m_curModelIndex;
+        }
+
     signals:
         void TagClicked(const Tag& tag);
         void UpdateGem(const QModelIndex& modelIndex);
         void UninstallGem(const QModelIndex& modelIndex);
+        void EditGem();
```

Comment:
If you pass in the modelIndex in `EditGem` as a parameter it looks like you won't need the `GetCurrentModelIndex` function at all